pyproject.toml

```diff
@@ -23,7 +23,14 @@ repository = "https://github.com/elixir-cloud-aai/tus-storagehandler"
 version = "0.1.0"
 
 [tool.poetry.dependencies]
-python = ">=3.11"
+python = ">=3.11,<4.0"
+foca = "^0.13.0"
+connexion = "2.14.2"
+multipart = "^0.2.5"
+python-multipart = "^0.0.9"
+python-dotenv = "^1.0.1"
```

Comment:
I dont think its a good idea to use dotenv package! firstly his should be developer dependent on how they wanna manage their env vars and lastly this would then give us another way to inject env variables into docker images and other container, I think we would be better off with just the standard way and not rely on `.env`. Also [check this post](https://stackoverflow.com/questions/67604414/why-i-should-not-use-dotenv-in-production-mode) out.